.github/workflows/ci.yml

```diff
@@ -88,6 +88,16 @@ jobs:
           - python: '3.8'  # <- not actually used
             pypy_nightly_branch: 'py3.8'
             extra_name: ', pypy 3.8 nightly'
+    continue-on-error: >-
```

Comment:
^ this is needed to match the currently deselected jobs in branch protection.

Reply:
How about adding a `not_required` attribute to the matrix?  I guess that would require dev Python versions get an `include:` entry.  Seems to me like a worthwhile trade for simplicity and explicitness here.

Reply:
I guess. I just didn't want to include too much changes in the PR.
FWIW GitHub's own docs feature the use of an `experimental` flag for this purpose.